src/mongo/unittest/unittest.h

```diff
@@ -231,6 +234,39 @@
     void _TEST_TYPE_NAME(FIXTURE_NAME, TEST_NAME)::_doTest()
 
 /**
+ * Construct a single benchmark named BENCHMARK_NAME that has access to a common class (a "fixture")
+ * named "FIXTURE_NAME".
+ *
+ * Usage:
+ *
+ * class FixtureClass : public mongo::unittest::Benchmark {
+ * protected:
+ *   int myVar;
+ *   void setUp() { myVar = 10; }
+ * };
+ *
+ * BENCHMARK(FixtureClass, TestThatUsesFixture) {
+ *     myVar += 1;
+ * }
+ */
+#define BENCHMARK(FIXTURE_NAME, BENCHMARK_NAME, ITERATIONS, FILE)                              \
+    class _TEST_TYPE_NAME(FIXTURE_NAME, BENCHMARK_NAME) : public FIXTURE_NAME {                \
+    private:                                                                                   \
+        NOINLINE_DECL virtual void _doTest();                                                  \
+        int numIterations() {                                                                  \
+            return ITERATIONS;                                                                 \
+        }                                                                                      \
+        std::string testName() {                                                               \
+            return QUOTE(BENCHMARK_NAME);                                                      \
```

Comment:
Any reason you can't just use #BENCHMARK_NAME here? If you can, delete QUOTE above.